guides/source/caching_with_rails.md

```diff
@@ -512,13 +512,19 @@ class ProductsController < ApplicationController
 end
 ```
 
-### A note on weak ETags
-
-Etags generated by Rails are weak by default. Weak etags allow semantically equivalent responses to have the same etags, even if their bodies do not match exactly. This is useful when we don't want the page to be regenerated for minor changes in response body. If you absolutely need to generate a strong etag, it can be assigned to the header directly.
```

Comment:
I'd keep this paragraph. Just remove the last sentence and discuss strong ETags separately.

Note that "ETag" has two leading capital letters. Let's change the inconsistent use of "Etag" and "etag" as well.